pkg/controllers/management/clustergc/cluster_scoped_gc.go

```diff
@@ -75,14 +75,19 @@ func (c *gcLifecycle) waitForNodeRemoval(cluster *v3.Cluster) error {
 	if err != nil && !errors.IsNotFound(err) {
 		return err
 	}
-
+	var wait bool
```

Comment:
Über nit: I would rename this waitForNodeDelete

Reply:
fixed.